production/src/test/java/io/jaegertracing/kubernetes/CassandraETest.java

```diff
@@ -20,9 +20,9 @@
 /**
  * @author Pavol Loffay
  */
-public class ProductionETest extends BaseETest {
+public class CassandraETest extends BaseETest {
 
-  @Ignore("dependency links returns 404 because of old Cassandra image")
+  @Ignore("It requires spark job")
```

Comment:
Same as above: if the only test is being ignored, can't you just remove the whole test class?

Reply:
No, test class extends base test class with other tests.